### DISCUSSION TITLE
Gotrue client SignOut()

### DISCUSSION THREAD

**Discussion by kienns1996** [supabase/supabase#22306] [Questions] [answered]
SignOut() function of Gotrue-csharp log out all sessions by default, is it possible to SignOut() with scope option "global", "local" or "others"?

**Comment by GaryAustin1** [comment, marked as answer]
If the option has not been added then no as it has to be done in the REST call.  
You should generate an issue the repository https://github.com/supabase-community/supabase-csharp/issues or as that is a community library add a PR to fix the signout code. 

Note I'm assuming it is not supported by your comment and a quick glance at the source. https://github.com/supabase-community/gotrue-csharp/blob/ddeb384d54454c762d437d4d6ba8f0f05e2f59c1/Gotrue/Api.cs#L497 which does not seem to pass the scope parameter.
